File: UVa/Contents/c002/c002.c
#include<stdio.h>

int main(void)
{
	int n;
	
	while(1)
	{
		scanf("%d", &n);
		
		if(!n)
			break;
		
		printf("f91(%d) = %d\n", n, (n <= 100) * 91 + (n > 100) * (n - 10));
	}	
		
	return 0;
}

File: UVa/Contents/c002/c002.py
import sys

for n in sys.stdin:
    n = int(n)

    if n == 0:
        break

    f = int(n <= 100) * 91 + int(n > 100) * (n - 10)
    print("f91(%d) = %d"  % (n, f))
